docs/cookbook/recipe_custom_view.rst

```diff
@@ -0,0 +1,122 @@
+Creating a Custom Admin View
+============================
+
+Sometimes, you may want to create an additional view in your Sonata admin
+but you do not have or need an Entity to connect it to. You also probably
+want to add it to the sidebar with the rest of your admin pages. Here we will
+present a full working example of how that is achievable.
+
+The recipe
+----------
+
+SonataAdmin provides a very straightforward way of adding your own custom
+view.
+
+To do this we need to:
+
+- create an admin class for your custom view;
```

Comment:
Why did you add `;` at the end of the lines?

Reply:
>greg0ire a day ago  Member
Please end bullet points with semi-colon, and the last one with a dot:
this;
that;
and finally, this.

/cc @greg0ire :smile:

Reply:
Because I asked him, and also that's also how you are supposed to format bullet lists, unless it contains sentences, in which case you should end them with a dot.

Reply:
Might be considered old-fashioned though: https://english.stackexchange.com/questions/75591/semicolons-at-the-end-of-list-items#75619